### PR TITLE
Allow using .env to configure database in development

### PR DESCRIPTION
**Pull request by ghost**
Allows configuring postgres host in development (useful for people using windows subsystem for linux which can't run postgres). Also adds sourcemaps for js debugging when in development.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -45,6 +45,9 @@ class Application < Rails::Application
 
     config.middleware.use Rack::Attack
     config.middleware.use Rack::Deflater
+    
+    # Enable sourcemaps for debugging during development
+    config.browserify_rails.source_map_environments << "development"
```
Comment (bradurani) [inline config/application.rb]: I feel like this would be better in a separate pull request

**Review by bradurani — CHANGES_REQUESTED** [review]

**Review by ashfurrow — CHANGES_REQUESTED** [review]
I've made these changes locally too, looks good 👍

Agree about moving the source map changes int their own PR, can you amend?

**Comment by mecab** [comment]
I think it is good PR and I would need it to be merged. If @rainyday is busy, can I raise PR (without the change about source map) instead?

**Review by ashfurrow — APPROVED** [review]
Looks great!

**Comment by mecab** [comment]
Cool, thanks!!

**Review by mjankowski — CHANGES_REQUESTED** [review]
Could people get the same benefit by using `ENV['DATABASE_URL']` (which rails already supports) - or is there something in this change which that doesn't provide and needs custom env?

**Comment by ashfurrow** [comment]
Hmm, I didn't know about `ENV['DATABASE_URL']`, if using it is compatible with this then maybe we should document that instead?

**Comment by Gargron** [comment]
DATABASE_URL env variable is preferable for this, you can set any options in it and it will override other settings. I don't want to have too many custom vars. Sorry
